torch_ttnn/backend.py

```diff
@@ -123,8 +130,7 @@ def aten_backend(
     gm.recompile()
 
     if option.metrics_path:
-        compiled_schema_list = metrics.collect_schema_from_nodes(gm.graph.nodes)
-        metrics.save_pickle(compiled_schema_list, option.metrics_path, "compiled-schema_list")
+        option.compiled_schema_list.extend(metrics.collect_schema_from_nodes(gm.graph.nodes))
```

Comment:
compiled schema contains schema for TT-NN ops?

Reply:
Yeah this part holds the graphs containing ttnn ops